collectors/python.d.plugin/riak/riak.chart.py

```diff
@@ -0,0 +1,528 @@
+# -*- coding: utf-8 -*-
+# Description: riak netdata python.d module
+#
+# See also:
+# https://docs.riak.com/riak/kv/latest/using/reference/statistics-monitoring/index.html
+
+from json import loads
+
+from bases.FrameworkServices.UrlService import UrlService
+
+# default module values (can be overridden per job in `config`)
+update_every = 2
+priority = 60000
+retries = 60
```

Comment:
`priority = 60000` and `retries = 60` are defaults, remove them

> update_every = 2

please add a comment why default update every is 2, and add it to the README to

Reply:
Documented in ba3df872d7357a877a26b25b669bcd8e04add5c6 with defaults removed.